src/drozer/modules/common/intent_filter.py

```diff
@@ -49,10 +49,15 @@ def from_attributes(cls, attrs):
             scheme = 'scheme' in attrs and attrs['scheme']
             host = 'host' in attrs and attrs['host']
             port = 'port' in attrs and attrs['port']
-            path = 'path' in attrs and attrs['path']
             mimetype = 'mimeType' in attrs and attrs['mimeType']
+            path = 'path' in attrs and attrs['path']
+            path_prefix = 'pathPrefix' in attrs and attrs['pathPrefix']
+            path_pattern = 'pathPattern' in attrs and attrs['pathPattern']
 
-            return cls(scheme, host, port, path, mimetype)
+            paths = filter(None, [path, path_prefix, path_pattern])
+            datas = map(lambda x: cls(scheme, host, port, x, mimetype), paths or [None])
+
+            return datas
 
         def __str__(self):
             return "%s://%s:%s%s (type: %s)" % (self.scheme or "*", self.host or "*", self.port or "*", self.path or "*", self.mimetype or "*")
```

Comment:
@mmmds, I think you've forgotten to update this function with the path_prefix. Could you review and test your changes please.

Reply:
@amarekano, I've checked this and it works correctly. Variable "path_prefix" is assigned to the field "path" in the constructor (as same as the "path_pattern").

Reply:
My apologies, you're right. For some reason I my local build didn't include your PR changes which is why I couldn't see any change in functionality. This PR is good to go :)

Thanks again for your contribution to drozer.